src/getJavaConfig.ts

```diff
@@ -28,6 +28,7 @@ export function getJavaConfig({
   const coursierPath = path.join(extensionPath, "./coursier");
   const extraEnv = {
     COURSIER_REPOSITORIES: customRepositories.join("|"),
+    JAVA_HOME: javaHome,
```

Comment:
You mentioned that 
> we will replace any existing JAVA_HOME with the one used for Metals

Just by setting this, is it really doing that?

Reply:
We add the extra env to the normal env and it replaces any duplicate variables.